### PR TITLE
Fix #2564 (was: circleci: test on 32bit arch)

### LINKED ISSUE #2564: packcc on 32bit arch
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=956349

### PR DESCRIPTION
**Pull request by masatake**
Initially, this change is for reproducing the issue #2564.
The steps for reproducing are reported by @k-takata.

### CODE REVIEW

**Comment by masatake** [comment]
```
Failed tests
============================================================
output-format-option/stdout-compare
omit-long-patterns-etags/stdout-compare
omit-long-patterns-etags/stderr-compare
pattern-length-limit/stdout-compare
output-file-resource-management/stdout-compare
```

It seems that there is a bug in code for etags output.

About units target, varlink related tests are failed as we expect.

**Comment by masatake** [comment]
```
/root/universal-ctags/Tmain/omit-long-patterns-etags.d/stderr-diff.txt
 

 
	--- /root/universal-ctags/Tmain/omit-long-patterns-etags.d/stderr-actual.txt	2020-06-27 16:44:59.057058341 +0000
 
	+++ ./Tmain/omit-long-patterns-etags.d/stderr-expected.txt	2020-06-27 16:43:48.427152549 +0000
 
	@@ -1 +0,0 @@
 
	-ctags: cannot open temporary file: /tmp/tags.29ZhTN : File exists
 ```

It seems that there is a bug in handling or manipulating a temporary file.
These failures are not reproduced in my local environment.

These are nothing to do with the original bug about packcc on 32 bit.
So till fixing the original issue, I will turn off the tmain target.

**Comment by masatake** [comment]
Issue #2564 is reproduced well!

**Comment by masatake** [comment]
The fixing of the packcc side is proposed at https://github.com/universal-ctags/packcc/pull/4.
Running pull-packcc.sh is needed before merging this.

File: Makefile.am [hunk 1]
```diff
@@ -146,9 +146,9 @@ packcc_verbose_0 = @echo PACKCC "    $@";
 PACKCC = $(top_builddir)/packcc$(EXEEXT)
 SUFFIXES += .peg
 .peg.c:
-	$(packcc_verbose)$(PACKCC) -o $(top_builddir)/peg/$(*F) "$<"
+	$(packcc_verbose)$(PACKCC) -i \"general.h\" -o $(top_builddir)/peg/$(*F) "$<"
 .peg.h:
-	$(packcc_verbose)$(PACKCC) -o $(top_builddir)/peg/$(*F) "$<"
+	$(packcc_verbose)$(PACKCC) -o \"general.h\" $(top_builddir)/peg/$(*F) "$<"
```
Comment (k-takata) [inline Makefile.am:151]: Is this correct?

Comment (masatake) [reply Makefile.am]: Thank you. Updated.

**Comment by masatake** [comment]
ubuntu20_32bit is failed at the master branch. Something goes wrong.
As far as reading the log message, the failure may be related to https://bugs.launchpad.net/ubuntu/+source/ubiquity/+bug/1871268 . More inspection is needed.

**Pull request merged**